.circleci/os/linux/dependencies.sh

```
@@ -7,16 +7,34 @@
 
 export PATH="$PATH:/home/qa/.yarn/bin"
 
-# web driver
+# Install web driver
+echo "Installing Playwright..."
 npx playwright install
 
-# verify env var
-echo $PUPPETEER_EXECUTABLE_PATH
+# Verify environment variable
+echo $PUPPETEER_EXECUTABLE_PATH
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)